test/alternator/util.py

```diff
@@ -224,6 +224,16 @@ def client_no_transform(client):
 def is_aws(dynamodb):
     return dynamodb.meta.client._endpoint.host.endswith('.amazonaws.com')
 
+# Return the AWS region name, or the Scylla data center name.
+def get_region(dynamodb):
+    if is_aws(dynamodb):
+        dc_name = dynamodb.meta.client.meta.region_name
```

Comment:
I wonder if we can change the way we configure our connection to have in the client.meta.region_name Scylla's real DC name, and not "us-east-1", when connected to Scylla. But it's not important now, this code looks good.